pyspecProcScripts/load_data.py

```diff
@@ -38,31 +38,25 @@ def proc_bruker_deut_IR_withecho_mancyc(s, fl=None):
         fl.next("IR prior to FTing ph")
         fl.image(s.C.setaxis("indirect", "#").set_units("indirect", "scan #"))
     s.ft(
-        ["ph1", "ph2"], unitary=True
-    )  # fourier transforming from phase cycle dim to coherence dimension
+        ["ph1", "ph2"]
+    )  # not doing a unitary FT in order to match the
+    #    coherence domain signal to its size in each transient
+    logging.info(
+        strm(
+            "I didn't know what to set the coherence pathway to so you need to manually do this by setting s.set_prop('coherence_pathway') = {'ph1':x, 'ph2':y}"
+        )
+    )
     s.reorder(["indirect", "t2"], first=False)
     if fl is not None:
         s_forplot = s.C
         fl.next("FT")
-        fl.image(
```

Comment:
you might need to merge from jf review actual so this doesn't show up as a diff.   jf review actual no longer differs here

I'm stopping reviewing this until this is fixed, because that seems wasteful